ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template2.kt

```diff
@@ -126,7 +140,7 @@ private fun SelectPackageButton(
     }
     val border = if (isSelected) null else BorderStroke(2.dp, colors.text1)
     Button(
-        modifier = Modifier.fillMaxWidth(),
+        modifier = Modifier.fillMaxWidth().then(childModifier),
```

Comment:
You could do 
```suggestion
        modifier = childModifier.fillMaxWidth(),
```
I think? We should see how it looks though.

Reply:
Yeah that's simpler 👍🏻